### PR TITLE
Update the HBase schema definition

### PR DESCRIPTION
**Pull request by JulienPeloton**
This PR updates the documentation on the HBase table schema. Several change have been made:
- versioning of the schema is done using versions of Fink broker and Fink science (`schema_<fb version>_<fs science>`)
- New entries (`d:mulens_class_<i>` and `d:roid`)
- Update `d:snn` name into `d:snnscore`
- One column family changed its name (`b` -> `fits`) to describe better the data.
- Row key definition is simpler: `objectId_timestamp` (instead of `objectId_timestamp_ra_dec`)

A data sample has been pushed in HBase (`test_portal_tiny.2`).

### CODE REVIEW

**Comment by JulienPeloton** [comment]
@saucam you would need to accept the invitation in this repo in order for me to select you as a reviewer.

**Comment by saucam** [comment]
Accepted

**Comment by hrivnac** [comment]
Concerning schema: I think, better would be that all binary columns belong to one family 'b' and schema would contain detailed description 'fits' (or 'image/fits'). Family should be more general then 'schema type'. Now it is the other way around, and it is confusing:
- It can eventually lead to explosion of of families.
- 'binary' as schema type is not sufficient.

Also, could you add a family (empty at the beginning) 'a' for 'annotations' ? It's just useful when one wants to tag some rows (as dead or something like that).

**Comment by JulienPeloton** [comment]
OK, concretely that would mean the following then?

```diff
- fits:cutoutDifference            timestamp=1591709722502, value=binary
- fits:cutoutScience               timestamp=1591709722502, value=binary
- fits:cutoutTemplate              timestamp=1591709722502, value=binary
+ b:cutoutDifference            timestamp=1591709722502, value=fits/image
+ b:cutoutScience               timestamp=1591709722502, value=fits/image
+ b:cutoutTemplate              timestamp=1591709722502, value=fits/image
```

I like the `a` family idea - I will add it.

**Comment by JulienPeloton** [comment]
I updated the schema with recent discussions:
- Column family for binary is `b`
- Type description for binary can be found in the schema row (e.g. `fits/image`)
- New column family `a` for annotations has been added. It is empty to start with.
- renamed all original nested names `a.b` as `a_b` (@hrivnac `cutout*` now becomes `cutout*_stampData`)

Updated data pushed in the table `test_portal_tiny.2` in the cluster.

@saucam @hrivnac let me know if that works.

File: docs/Schema-Model.md [hunk 1]
```diff
@@ -2,136 +2,148 @@
...
+- `i`: original alert information,
+- `d`: added values by Fink,
+- `b`: binary data (e.g. binary gzipped FITS image),
+- `a`: for annotations (initially empty).
 
-Note that this schema is subject to evolution.
+A single alert row has the following fields in HBase
 ```
-key: ZTF19acmbtud_2458789.0279514_128.2831039_15.3663548
+key: schema_0.6.0_0.3.2
```
Comment (saucam) [inline docs/Schema-Model.md:13]: Could you explain how we generate this key?

**Review by saucam — APPROVED** [review]

**Review by saucam — APPROVED** [review]

File: docs/Schema-Model.md (hunk 1 above)
Comment (saucam) [inline docs/Schema-Model.md:13]: This row key must be unique for every alert no?

Comment (JulienPeloton) [reply docs/Schema-Model.md:13]: The script that pushes data to the HBase table has two parts:
1. push alert data into HBase table (rowkey `<objectId>_<timestamp>`): https://github.com/astrolabsoftware/fink-broker/blob/92c8988e795e6454602d237ed255e863b48885a1/bin/science_archival.py#L95
2. push a **single schema row** into HBase table (rowkey `schema_<fink broker version>_<fink science version>`): https://github.com/astrolabsoftware/fink-broker/blob/92c8988e795e6454602d237ed255e863b48885a1/bin/science_archival.py#L123

The schema row is a unique row describing data types (unlike other rows that are the data itself).

Comment (saucam) [reply docs/Schema-Model.md:13]: ah ok thanks ! this document is describing the alert to hbase transformation though, maybe I can add this information as well

Comment (JulienPeloton) [reply docs/Schema-Model.md:13]: OK I will let you reshape the text after the merge.

**Pull request merged**